frontend/www/js/omegaup/components/problem/CollectionDetails.vue

```diff
@@ -1,20 +1,63 @@
 <template>
   <div>
     <h1 class="card-title">{{ title }}</h1>
+    <div class="row">
+      <div class="col col-md-4">
+        <div>
+          <omegaup-problem-filter-tags
+            :tags="collectionTags"
+            :public-tags="publicTags"
+            :selected-tags="selectedTags"
+            @add-tag="addTag"
+            @check="(selectedTags) => check(selectedTags)"
```

Comment:
```suggestion
            @add-tag="addTag"
            v-model="selectedTags"
```

esto es equivalente y ya no habría necesidad del `check()`. aquí la documentación de cómo debe ir en el otro lado: https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components

Reply:
@lhchavez  No consigo hacer que esto funcione. Al poner eso y del lado del componente `v-bind:value="tag" v-on:input="$emit('input', $event.target.value)"` como dice la documentación, el valor se sobrescribe siempre y no se añade o quita del arreglo al marcar o desmarcar el checkbox. ¿Cómo se podría arreglar? llevo mucho intentando.

Reply:
@pabo99 puedes ayudar?

Reply:
Claro, en un momento reviso.